control-plane/pkg/reconciler/channel/channel.go

```diff
@@ -93,7 +93,7 @@ type Reconciler struct {
 	ServiceLister      corelisters.ServiceLister
 	SubscriptionLister messaginglisters.SubscriptionLister
 
-	Prober prober.Prober
+	Prober prober.NewProber
```

Comment:
In this file, you will also have to edit lines 347-352, where you create `proberAddressable` so that it is of type `prober.NewAddressable`. See the following for how that is being done in the broker: https://github.com/knative-sandbox/eventing-kafka-broker/blob/04f560e4ebd0b7d157911ebe7b31488772b0c33c/control-plane/pkg/reconciler/broker/broker.go#L266-L272